test.sh

```diff
@@ -1,12 +1,16 @@
 #!/usr/bin/env bash
 
-set -eux
+set -ex
 
 export SOURCE_DIR="$PWD/example"
 export PDFium_DIR="$PWD/staging"
 
+[ "$TARGET_CPU" != "" ] && export CMAKE_OSX_ARCHITECTURES="$TARGET_CPU"
+
 mkdir build
 cd build
 cmake "$SOURCE_DIR"
 cmake --build .
-ctest --output-on-failure .
\ No newline at end of file
+if [ "$TARGET_CPU" == "" ]; then
```

Comment:
Can you use `[ "$TARGET_CPU" != "" ] && ...` as above?

Reply:
I tried that before, but because it's the last line in the script the script would return a non zero return value (when the TARGET_CPU isn't empty), which would trigger a travis failure.